File: hotel_ratings/v2/get/node/hotel_ratings.js
var Amadeus = require("amadeus");
var amadeus = new Amadeus({
  clientId: 'YOUR_API_KEY',
  clientSecret: 'YOUR_API_SECRET'
});

// What travelers think about this hotel?
amadeus.eReputation.hotelSentiments.get({
  hotelIds: 'ADNYCCTB'
}).then(function (response) {
  console.log(response);
}).catch(function (response) {
  console.error(response);
});

File: hotel_ratings/v2/get/python/hotel_ratings.py
# Install the Python library from https://pypi.org/project/amadeus
from amadeus import Client, ResponseError

amadeus = Client(
    client_id='YOUR_AMADEUS_API_KEY',
    client_secret='YOUR_AMADEUS_API_SECRET'
)

try:
    '''
    What travelers think about this hotel?
    '''
    response = amadeus.e_reputation.hotel_sentiments.get(hotelIds = 'ADNYCCTB')
    print(response.data)
except ResponseError as error:
    raise error
